### PR TITLE
Ewerton - Celo Sage - How to tokenize physical assets on Celo

### PR DESCRIPTION
**Pull request by guizostudios**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [X] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [X] The title of the Pull Request matches the title of my tutorial.
- [X] I've followed the tutorial template structure.
- [X] I've proofread the tutorial for spelling and grammar mistakes.
- [X] I've thoroughly explained the code, with inline code comments and text explanations.
- [X] I've run a linter & prettifier to format my code blocks.
- [X] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+
+![header](../../src/data-tutorials/showcase/beginner/how-to-tokenize.png)
+
+## What is tokenization?
+
+Tokenization is the process of creating a digital token that represents a real-world asset. This token can be traded on a blockchain, which is a distributed ledger that allows for the secure and transparent transfer of ownership of the asset.
+
+Tokenization has the **potential to revolutionize** the way we own and trade assets by making it easier to buy, sell, and transfer ownership of a wide range of assets, including real estate, precious metals, and even fine art. It can also help to increase liquidity and reduce the costs associated with trading assets, as the use of smart contracts and blockchain technology can streamline the transaction process.
+
+On July 14, 2022, Itaú, a big bank in Brazil, launched its own cryptocurrency unit, which aims to tokenize fixed income securities and shares traded in the market in order to make alternatives that are currently restricted to high net worth investors more accessible.
```
Comment (Julius170) [inline blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md:23]: On July 14, 2022, Itaú, a big bank in Brazil, launched its cryptocurrency unit, which aims to tokenize fixed-income securities and shares traded in the market to make alternatives that are currently restricted to high-net-worth investors more accessible.

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+
+In addition, in the second half of July, the educational platform Pravaler became the first company in the sector to issue tokenized debentures in the local market, raising around U$ 12 million in the operation.
+
+In the Brazil market, there are already companies that have been operating in the area of tokenization of real assets for some years, converting them into NFTs, which stands for Non-Fungible Token, such as MB Tokens, a subsidiary of Mercado Bitcoin focused on the segment, and Hurst Capital.
```
Comment (Julius170) [inline blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md:27]: In the Brazil market, there are already companies that have been operating in the area of tokenization of real assets for some years, converting them into NFTs, which stands for Non-Fungible Token, such as MB Tokens, a subsidiary of Mercado Bitcoin focused on the segment and Hurst Capital.

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+
+These companies tokenize and offer investors in small fractions anything that can be imagined, as long as it has some real asset behind it that serves as collateral - from consortium quotas to energy marketing contracts, passing through works of art, music royalties, and even the sale of soccer stars like Neymar and Gabriel Barbosa.
+
+These contracts of various kinds are registered on the blockchain network and fractionated, and then offered at affordable prices on cryptocurrency trading platforms to investors.
+
+The **[Celo hackathon to tokenize government assets in Brazil](https://patrimoniodauniaonaweb3.hackerearth.com/)** was an exciting initiative that aims to leverage the power of blockchain technology to promote transparency and efficiency in the management of public assets.
+
+The tokenization of government assets has the potential to transform the way public assets are managed around the world, by enabling greater transparency and accountability in the ownership, transfer, and management of these assets. By creating digital tokens that represent these assets, it becomes possible to track their ownership and transaction history in a secure and transparent way.
```
Comment (Julius170) [inline blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md:35]: The tokenization of government assets has the potential to transform the way public assets are managed around the world, by enabling greater transparency and accountability in the ownership, transfer, and management of these assets. By creating digital tokens that represent these assets, it becomes possible to track their ownership and transaction history securely and transparently.

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md [hunk 4]
```diff
@@ -0,0 +1,300 @@
...
+
+## Mechanics of on-chain asset tokenization:
+
+**Build the ecosystem:** Service providers ensure that technology integration is seamlessly done with a large ecosystem of vendors so that issuers can create an end-to-end customized solution (e.g., custody, KYC/AML provider, legal, marketing, advisory, tax, etc.).
+
+**Register the underlying asset and set up the token:** The asset is registered and made available for tokenization on the blockchain network. Once the raise is structured and the ecosystem is mapped, service providers enable issuers to configure their tokens in a simple and intuitive way, regardless of the type of asset. The number of assets to be registered on the platform is a decision made in alignment between the issuer and the platform provider.
```
Comment (Julius170) [inline blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md:43]: **Register the underlying asset and set up the token:** The asset is registered and made available for tokenization on the blockchain network. Once the raise is structured and the ecosystem is mapped, service providers enable issuers to configure their tokens simply and intuitively, regardless of the type of asset. The number of assets to be registered on the platform is a decision made in alignment between the issuer and the platform provider.

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md [hunk 5]
```diff
@@ -0,0 +1,300 @@
...
+
+**Verification:** The ownership and authenticity of the asset are verified through the use of documents, certificates, or other forms of evidence.
+
+**Trading:** The asset is listed for sale on a marketplace or exchange that supports the trading of tokenized assets. Potential buyers can purchase the asset using the token that represents it.
+
+**Transfer:** When the asset is sold, the token representing the asset is transferred from the seller's wallet to the buyer's wallet. This transfer is recorded on the blockchain and is secured by the smart contract that was created during the creation stage.
+
+**Settlement:** After the transfer is complete, the asset is considered to be in the possession of the buyer. The settlement process may involve the transfer of physical ownership of the asset, or it may simply involve the transfer of ownership rights as represented by the token.
+
+**Redeemed:** In some cases, an asset may be redeemed by the original owner or by a third party. This process typically involves the transfer of the token back to the original owner or to the third party, and may be accompanied by the transfer of physical ownership or ownership rights.
```
Comment (Julius170) [inline blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md:66]: **Redeemed:** In some cases, an asset may be redeemed by the original owner or by a third party. This process typically involves the transfer of the token back to the original owner or the third party and may be accompanied by the transfer of physical ownership or ownership rights.

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md [hunk 6]
```diff
@@ -0,0 +1,300 @@
...
+
+It is important to note that the specific details of the life cycle of a tokenized asset will depend on the type of asset being tokenized and the terms of the smart contract that represents it.
+
+## Tokenize a physical asset on Celo
+
+To tokenize a physical asset on the Celo blockchain, you will need to follow these steps:
+
+*Determine the type of asset* you want to tokenize and ensure that it is allowed on the Celo platform. Some examples of assets that can be tokenized on Celo include real estate, precious metals, and works of art.
+
+*Choose the appropriate Ethereum Request for Comment (ERC) standard* for representing your asset on the blockchain. There are several different ERC standards that can be used for tokenizing assets, including ERC-721 and ERC-20.
```
Comment (Julius170) [inline blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md:76]: *Choose the appropriate Ethereum Request for Comment (ERC) standard* for representing your asset on the blockchain. Several different ERC standards can be used for tokenizing assets, including ERC-721 and ERC-20.

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md [hunk 7]
```diff
@@ -0,0 +1,300 @@
...
+
+*Create a smart contract on the Celo blockchain* that represents the asset and defines its ownership and transfer rules. This smart contract should specify the conditions under which the asset can be bought, sold, or transferred, as well as any restrictions or limitations on these actions.
+
+## Different types of ERC can be used to represent different types of assets on Celo.
+
+Ethereum Request for Comment (ERC) standards are technical specifications that define how tokens can be created and used on the Celo blockchain. There are several different ERC standards that can be used for tokenization, including ERC-20, ERC-721, ERC-1400, ERC-3643, and ERC-1404.
```
Comment (Julius170) [inline blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md:82]: Ethereum Request for Comment (ERC) standards are technical specifications that define how tokens can be created and used on the Celo blockchain. Several different ERC standards can be used for tokenization, including ERC-20, ERC-721, ERC-1400, ERC-3643, and ERC-1404.

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md [hunk 8]
```diff
@@ -0,0 +1,300 @@
...
+
+**ERC-20** is a widely used standard for creating tokens that represent interchangeable assets, such as commodities or currencies. ERC-20 tokens are created using smart contracts and are stored in Celo wallets. They can be traded on exchanges and used in a variety of applications, such as fundraising through initial coin offerings or representing ownership in a company or project.
+
+OpenZeppelin provides a powerful set of tools to help you create your own token on Ethereum, including the popular ERC20 standard. You can see it on [https://docs.openzeppelin.com/contracts/4.x/wizard](https://docs.openzeppelin.com/contracts/4.x/wizard)
```
Comment (Julius170) [inline blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md:86]: OpenZeppelin provides a powerful set of tools to help you create your token on Ethereum, including the popular ERC20 standard. You can see it on [https://docs.openzeppelin.com/contracts/4.x/wizard](https://docs.openzeppelin.com/contracts/4.x/wizard)

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md [hunk 9]
```diff
@@ -0,0 +1,300 @@
...
+    
+*8.  constructor() ERC20("Tokenization of Physical Asset", "Tokenize") {:* This is the constructor for the contract, which is called when the contract is first deployed. It initializes the token with a name of "Tokenization of Physical Asset" and a symbol of "Tokenize". The ERC20 contract's constructor is also called, which sets up some initial variables.
+    
+*9.  _mint(msg.sender, 100 * 10 ** decimals());:* This function call mints 100 tokens to the address that deployed the contract, which is represented by msg.sender. The number is multiplied by 10 ** decimals(), which sets the number of decimal places for the token. By default, the ERC20 standard has 18 decimal places, so this call will mint 100 tokens with 18 decimal places.
+    
+*10.  function pause() public onlyOwner {:* This function is used to pause the token contract. It can only be called by the contract owner.
+    
+*11.  _pause();:* This is the function call that actually pauses the token contract. It is provided by the Pausable contract.
```
Comment (Julius170) [inline blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md:150]: *11. _pause();:* This is the function call that pauses the token contract. It is provided by the Pausable contract.

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md [hunk 10]
```diff
@@ -0,0 +1,300 @@
...
+    
+*12.  function unpause() public onlyOwner {:* This function is used to unpause the token contract. It can only be called by the contract owner.
+    
+*13.  _unpause();:* This is the function call that actually unpauses the token contract. It is provided by the Pausable contract.
```
Comment (Julius170) [inline blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md:154]: *13. _unpause(); : * This is the function call that unpauses the token contract. It is provided by the Pausable contract.

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md [hunk 11]
```diff
@@ -0,0 +1,300 @@
...
+    
+*14.  function mint(address to, uint256 amount) public onlyOwner {:* This function is used to mint new tokens to a specific address. It can only be called by the contract owner.
+    
+*15.  function _beforeTokenTransfer(address from, address to, uint256 amount) internal whenNotPaused override {* This is a function that is defined in the TokenizationOfPhysicalAsset contract. It overrides the _beforeTokenTransfer function inherited from the ERC20 contract, which is called before every transfer of tokens.
+    
+  
+
+**ERC-721** is a standard for creating non-fungible tokens, which are unique tokens that represent one-of-a-kind assets. ERC-721 tokens are also created using smart contracts and are stored in Ethereum wallets. They can be traded on exchanges and used to represent ownership of unique assets.
+
+To tokenize physical assets on Celo, you can use a standard called ERC-721.You can also start your contract with Openzeppelin and then customize for yours needs.
```
Comment (Julius170) [inline blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md:164]: To tokenize physical assets on Celo, you can use a standard called ERC-721. You can also start your contract with Openzeppelin and then customize it for your needs.

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md [hunk 12]
```diff
@@ -0,0 +1,300 @@
...
+    
+*-   import "@openzeppelin/contracts/security/Pausable.sol";:* This import statement brings in the Pausable contract from OpenZeppelin. This allows the contract owner to pause and unpause token transfers in case of emergencies.
+    
+*-   import "@openzeppelin/contracts/access/Ownable.sol";:* This import statement brings in the Ownable contract from OpenZeppelin. This contract implements a basic access control mechanism that restricts access to certain functions to the contract owner.
+    
+*-   import "@openzeppelin/contracts/token/ERC721/extensions/ERC721Burnable.sol";:* This is another import statement that brings in the ERC721Burnable contract from OpenZeppelin. This extension adds the ability to burn or destroy tokens.
+    
+*-   import "@openzeppelin/contracts/utils/Counters.sol";:* This import statement brings in the Counters library from OpenZeppelin. Counters is used to keep track of the token IDs.
```
Comment (Julius170) [inline blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md:238]: *- import "@openzeppelin/contracts/utils/Counters.sol";:* This import statement brings in the Counters library from OpenZeppelin. Counters are used to keep track of the token IDs.

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md [hunk 13]
```diff
@@ -0,0 +1,300 @@
...
+    
+
+The contract itself is called TokenizationNFT and inherits from several of the imported contracts, including ERC721, ERC721URIStorage, Pausable, Ownable, and ERC721Burnable.
+
+The using Counters for Counters.Counter; statement is used to create a Counters.Counter variable that will keep track of the token IDs.
```
Comment (Julius170) [inline blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md:243]: The using Counters for Counters.Counter; statement is used to create a Counters. The counter variable will keep track of the token IDs.

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md [hunk 14]
```diff
@@ -0,0 +1,300 @@
...
+
+The constructor function sets the name and symbol of the token.
+
+The _baseURI function returns the base URI for the metadata associated with the token.
+
+The pause and unpause functions allow the contract owner to pause and unpause token transfers.
+
+The safeMint function is used to mint a new token and associate it with the specified uri. The to address specifies who the token will be sent to.
```
Comment (Julius170) [inline blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md:251]: The safeMint function is used to mint a new token and associate it with the specified uri. The address specifies to who the token will be sent.

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md [hunk 15]
```diff
@@ -0,0 +1,300 @@
...
+
+Tokenization using these ERC standards allows for the creation and trade of a wide range of assets on the Celo blockchain. By using smart contracts to define the ownership and transfer rules for tokens, tokenization can increase the liquidity and accessibility of assets, making it easier to buy, sell, and transfer ownership.
+
+## Conclusion
+
+In conclusion, tokenization of assets on the **Celo blockchain is a powerful tool** that enables the creation of digital representations of real-world assets. This process involves breaking down an asset into smaller units or tokens that can be traded and transferred on the blockchain. The mechanics of on-chain asset tokenization involve the creation of a smart contract that defines the rules for the issuance, management, and transfer of the tokens.
```
Comment (Julius170) [inline blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md:288]: In conclusion, the tokenization of assets on the **Celo blockchain is a powerful tool** that enables the creation of digital representations of real-world assets. This process involves breaking down an asset into smaller units or tokens that can be traded and transferred on the blockchain. The mechanics of on-chain asset tokenization involve the creation of a smart contract that defines the rules for the issuance, management, and transfer of the tokens.

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md [hunk 16]
```diff
@@ -0,0 +1,300 @@
...
+
+The **life cycle of an asset** on the Celo blockchain starts with the creation of the asset token and its issuance to the owner of the underlying asset. The tokens can then be traded and transferred among users, with each transaction being recorded on the blockchain. When the underlying asset is sold or otherwise disposed of, the tokens are burned and removed from circulation.
+
+Various types of assets can be tokenized on the Celo blockchain, including traditional assets such as real estate, stocks, and bonds, as well as non-traditional assets such as intellectual property and carbon credits. Tokenization makes these assets more accessible to a wider range of investors, as they can be bought and sold in smaller units and traded more easily than traditional assets.
+
+Overall, **on-chain asset tokenization** on the Celo blockchain provides a secure, transparent, and efficient way to digitize assets and create new opportunities for investment and innovation.
+
+## Sobre o Autor
+
+I am a serial entrepreneur, founder of Guizo Studios and always available to help the Celo ecosystem.
```
Comment (Julius170) [inline blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md:298]: I am a serial entrepreneur, founder of Guizo Studios, and always available to help the Celo ecosystem.

**Review by Julius170 — COMMENTED** [review]
Hi @guizostudios I've made some suggestions 
you can now make changes based on the suggestions

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md (hunk 1 above)
Comment (guizostudios) [reply blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md]: Done

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md (hunk 2 above)
Comment (guizostudios) [reply blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md]: Done

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md (hunk 4 above)
Comment (guizostudios) [reply blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md]: Done

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md (hunk 5 above)
Comment (guizostudios) [reply blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md]: Done

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md (hunk 6 above)
Comment (guizostudios) [reply blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md]: Done

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md (hunk 7 above)
Comment (guizostudios) [reply blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md]: Done

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md (hunk 8 above)
Comment (guizostudios) [reply blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md]: Done

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md (hunk 9 above)
Comment (guizostudios) [reply blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md]: Done

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md (hunk 10 above)
Comment (guizostudios) [reply blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md]: Done

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md (hunk 11 above)
Comment (guizostudios) [reply blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md]: Done

**Comment by guizostudios** [comment]
Done

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md [hunk 17]
```diff
@@ -73,219 +73,196 @@ To tokenize a physical asset on the Celo blockchain, you will need to follow the
...
 
 **ERC-20** is a widely used standard for creating tokens that represent interchangeable assets, such as commodities or currencies. ERC-20 tokens are created using smart contracts and are stored in Celo wallets. They can be traded on exchanges and used in a variety of applications, such as fundraising through initial coin offerings or representing ownership in a company or project.
 
-OpenZeppelin provides a powerful set of tools to help you create your own token on Ethereum, including the popular ERC20 standard. You can see it on [https://docs.openzeppelin.com/contracts/4.x/wizard](https://docs.openzeppelin.com/contracts/4.x/wizard)
+OpenZeppelin provides a powerful set of tools to help you create your token on Ethereum, including the popular ERC20 standard. You can see it on [https://docs.openzeppelin.com/contracts/4.x/wizard](https://docs.openzeppelin.com/contracts/4.x/wizard)
+Contributor
 
 Here is an example of ERC20
 ```
```
Comment (Julius170) [inline blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md:90]: add the language type after the backslash for example ```solidity...

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md [hunk 18]
```diff
@@ -73,219 +73,196 @@ To tokenize a physical asset on the Celo blockchain, you will need to follow the
...
+
+
 
 **ERC-721** is a standard for creating non-fungible tokens, which are unique tokens that represent one-of-a-kind assets. ERC-721 tokens are also created using smart contracts and are stored in Ethereum wallets. They can be traded on exchanges and used to represent ownership of unique assets.
 
-To tokenize physical assets on Celo, you can use a standard called ERC-721.You can also start your contract with Openzeppelin and then customize for yours needs.
+To tokenize physical assets on Celo, you can use a standard called ERC-721. You can also start your contract with Openzeppelin and then customize it for your needs.
+
 
 ```
```
Comment (Julius170) [inline blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md:158]: solidity here too

**Review by Julius170 — COMMENTED** [review]
A little more changes and we're onto publishing

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md (hunk 12 above)
Comment (guizostudios) [reply blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md]: Done

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md (hunk 14 above)
Comment (guizostudios) [reply blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md]: Done

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md (hunk 15 above)
Comment (guizostudios) [reply blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md]: Done

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md (hunk 17 above)
Comment (guizostudios) [reply blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md]: Done

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md (hunk 18 above)
Comment (guizostudios) [reply blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md]: Done

File: blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md (hunk 16 above)
Comment (guizostudios) [reply blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md]: Done

**Pull request merged**